packages/ui/src/components/va-data-table/types/index.ts

```diff
@@ -66,6 +68,6 @@ export interface TableRow {
 
 export type TFilterMethod = (source: any) => boolean
 
-export type TSortingOrder = 'asc' | 'desc' | null
+export type TSortingOrder = 'asc' | 'desc' | null | string
```

Comment:
Does `any string` have a purpose here? On first look, it can't have any value exept `'asc' | 'desc' | null`.